Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/Snapshots/ComplexCalculator_Subtract_Generated.g.cs

```diff
@@ -15,7 +15,7 @@ public ComplexCalculator_Subtract_Generated()
             complexCalculator = new ComplexCalculator();
         }
 
-        public APIGatewayHttpApiV2ProxyResponse Subtract(APIGatewayHttpApiV2ProxyRequest request, ILambdaContext context)
+        public Amazon.Lambda.APIGatewayEvents.APIGatewayHttpApiV2ProxyResponse Subtract(Amazon.Lambda.APIGatewayEvents.APIGatewayHttpApiV2ProxyRequest request, Amazon.Lambda.Core.ILambdaContext context)
```

Comment:
Can we replace the the `Amazon.Lambda.APIGatewayEvents` prefix with a `using` statement ?

Reply:
The idea behind this is to use full names in order not to miss any using statement because request and response will vary based on the events. It is easier to maintain them using full name then figuring what using statement is required and include it.